package.json

```diff
@@ -8,12 +8,17 @@
     "node": ">=6.13.0"
   },
   "scripts": {
-    "build": "webpack --config ./webpack.config.js --watch"
+    "build": "webpack --config ./webpack.config.js --watch",
```

Comment:
@harrisonywu @krise3 I don't think the `--config ./webpack.config.js` option is necessary so long as the config file is in the root directory.